code/linear-regression/src/Components/GradientDescent.svelte

```diff
@@ -76,12 +76,12 @@
 
       Our goal is to find the coefficients, {@html katexify(`\\beta_0`, false)} and
       {@html katexify(`\\beta_1`, false)}, to minimize the error function. To do
-      this, we'll use the gradient, which represents the direction where that
-      function is increasing, and the rate at which it is increasing. Gradient
-      descent works by taking steps in the direction oppositie of where our
-      error function is increasing, proportional to the rate of change. To find
-      the coefficients that minimize First, we calculate the derivatives of our
-      loss function, MSE:
+      this, we'll use the gradient, which represents the direction that the
```

Comment:
I prefer a mix of the old and new text. Let's rewrite this as:

this, we'll use the gradient, which represents the direction that the function is increasing, and the rate at which it is increasing. Since we want to find the minimum of this function, we can go in the opposite direction of where it's increasing. This is exactly what Gradient Descent does, it works by taking steps in the direction opposite of where our error function is increasing, proportional to the rate of change. To find the coefficients that minimize the function, we first calculate the derivatives of our